src/main/java/dev/isxander/controlify/controller/GenericControllerConfig.java

```diff
@@ -26,6 +26,7 @@ public class GenericControllerConfig implements ConfigClass {
     public float chatKeyboardHeight = showOnScreenKeyboard ? 0.5f : 0f;
 
     public ResourceLocation[] radialActions = new ResourceLocation[8];
+    public int radialButtonFocusTimeoutMs = 1000;
```

Comment:
This should be stored in ticks, then increment the slider in 50ms intervals. Then in the binding, divide by 20.

Reply:
Because values in config should be optimised for usage, not for user configurement, this reduces complexity on usage

Reply:
And if the user later on changes tps to something other than 20, you are fine with the timing being out of sync?

Reply:
See if you fancy the latest commit better.